tests/WorkflowEnvironment.cs

```diff
@@ -0,0 +1,36 @@
+namespace TemporalioSamples.Tests;
+
+using System;
+using Temporalio.Client;
+using Xunit;
+
+public class WorkflowEnvironment : IAsyncLifetime
+{
+    private Temporalio.Testing.WorkflowEnvironment? env;
+
+    public ITemporalClient Client =>
+        env?.Client ?? throw new InvalidOperationException("Environment not created");
+
+    public async Task InitializeAsync()
+    {
+        env = await Temporalio.Testing.WorkflowEnvironment.StartLocalAsync(new()
+        {
+            DevServerOptions = new()
+            {
+                ExtraArgs =
+                [
+                    "--dynamic-config-value",
+                    "frontend.enableUpdateWorkflowExecution=true",
```

Comment:
I wonder if we should enable stuff like this when go to public preview?

Reply:
Yes, I think we should according to the implied opt-in vs opt-out nature of release stages at https://docs.temporal.io/evaluate/release-stages. But of course that is a server thing and not related to this PR.